src/uct/base/uct_component.h

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2019.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+
+#ifndef UCT_COMPONENT_H_
+#define UCT_COMPONENT_H_
+
+#include <uct/api/uct.h>
+#include <ucs/config/parser.h>
+#include <ucs/datastruct/list.h>
+
+
+extern ucs_list_link_t uct_md_components_list;
+
+
+typedef struct uct_md_component uct_md_component_t;
+struct uct_md_component {
```

Comment:
since this is moved, maybe rename?

Reply:
will be done in subsequent PR, since this rename will involve changes in multiple UCT transports